### PR TITLE
[PTT-116] add endpoint payments

### PR DESCRIPTION
**Pull request by filma80**
This PR proposes to add endpoints for transform the csv and load into DB.
#### List of Changes
-Create new controller PaymentsController
-Create new service PaymentService and IuvCodeGenerator
-Create  the entity for the structure of DB
-Create all the DTOs
-Create the converters from model to dto and from model to model

#### Motivation and Context

The new endpoint is necessary for save into the DB the new structure from the csv file
#### How Has This Been Tested?

-Suite of integraiton tests
-Manual Tests via Swagger-UI
-The failure cases will be tested in a new PR
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -50,7 +50,27 @@ services:
...
+      - postgres
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://postgres:5432/paymen
+      - SPRING_DATASOURCE_USERNAME=postgres-user-test
+      - SPRING_DATASOURCE_PASSWORD=postgres-pwd-test
+      - SPRING_JPA_HIBERNATE_DDL_AUTO=update
+      - SPRING_CORS_ORIGINS=http://localhost:8083,http://payments:8085
+      - TAXONOMY_PRIMARY=9/0101101IM/
+      - TAXONOMY_SECONDARY=9/0201133IM/
+      - SERVICEMANAGEMENT_PATH==http://localhost:8083/
```
Comment (GiovanniMancini) [inline docker-compose.yml:73]: This line is apprently wrong, as it uses `==` operator. Could you check please?

File: upload-payments/src/main/java/it/gov/pagopa/hubpa/uploadpayments/mapping/ConvertUploadCsvModelToPaymentsModel.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
     
     private static final String TAXONOMY_PRIMARY=System.getenv().get("TAXONOMY_PRIMARY");
```
Comment (GiovanniMancini) [inline upload-payments/src/main/java/it/gov/pagopa/hubpa/uploadpayments/mapping/ConvertUploadCsvModelToPaymentsModel.java:26]: Is taxonomy a property of the service? In such case I dont think it's a good idea set them here as env variables

**Review by GiovanniMancini — CHANGES_REQUESTED** [review]
We should check better how to model the taxonomy. Please resolve conflict with the main branch

File: upload-payments/src/main/java/it/gov/pagopa/hubpa/uploadpayments/mapping/ConvertUploadCsvModelToPaymentsModel.java (hunk 2 above)
Comment (filma80) [reply upload-payments/src/main/java/it/gov/pagopa/hubpa/uploadpayments/mapping/ConvertUploadCsvModelToPaymentsModel.java:26]: I'll open a new task for manage it

File: docker-compose.yml (hunk 1 above)
Comment (filma80) [reply docker-compose.yml:73]: i correct it

File: docker-compose.yml [hunk 3]
```diff
@@ -50,11 +50,32 @@ services:
       - SPRING_DATASOURCE_PASSWORD=postgres-pwd-test
       - SPRING_JPA_HIBERNATE_DDL_AUTO=update
       - SPRING_CORS_ORIGINS=http://localhost:8083,http://upload-payments:8084
-      - QUEUE_NAME=job_queue
+      - QUEUE_NAME=queue-name
       - JMS_AZURA_CONNECTION_STRING=
```
Comment (GiovanniMancini) [inline docker-compose.yml:54]: Please rename this variables as JMS_AZURE_xxx

Comment (filma80) [reply docker-compose.yml]: done

File: payments/src/main/resources/application.properties [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+spring.datasource.username=${SPRING_DATASOURCE_USERNAME}
+spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
+
+# The SQL dialect makes Hibernate generate better SQL for the chosen database
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+
+# Hibernate ddl auto (create, create-drop, validate, update)
+spring.jpa.hibernate.ddl-auto = ${SPRING_JPA_HIBERNATE_DDL_AUTO}
+
+iuv.generate.auxDigit=3
```
Comment (GiovanniMancini) [inline payments/src/main/resources/application.properties:12]: Is it right to have here static values?

**Review by GiovanniMancini — CHANGES_REQUESTED** [review]
I've seen you have merged main, but we still have conflicts

File: payments/src/main/resources/application.properties (hunk 4 above)
Comment (filma80) [reply payments/src/main/resources/application.properties]: done

**Comment by GiovanniMancini** [comment]
@filma80 the conflict was related to a residual file in JOB microservice. I got rid of it. Anyway we can add it at a later time if it proves to be needed

**Pull request merged**